memtier_benchmark.cpp

```diff
@@ -707,6 +707,9 @@ static int config_parse_args(int argc, char *argv[], struct benchmark_config *cf
         }
     }
 
+    if (cfg->verify_only && cfg->crc_verify)
+	    cfg->data_verify = 0;
```

Comment:
can you please explain how this code allow verify-only with crc-verify?

Reply:
btw, what is verify only mode?

Reply:
verify only mode assumes that previously verify was run .. verify only will read all keys and verify data integrity on those keys